src/getting-started.md

```diff
@@ -77,7 +77,7 @@ Depending on mining device you do run, you may have to adjust `tproxy-config-loc
 
 Download a release from Sjors' fork of Bitcoin Core from https://github.com/Sjors/bitcoin/releases
 
-Edit the `bitcoin.conf` file by adding:
+Edit the `bitcoin.conf` file stored in `~/.bitcoin/` by adding:
 ```bash
 server=1
 rpcuser=username
```

Comment:
if we're touching the user's global config file, we shouldn't impose these changes for every possible network configuration (maybe they're already running mainnet with some custom rpcuser/rpcpassword)

```suggestion
Edit the `bitcoin.conf` file stored in `~/.bitcoin/` by adding:
```bash
[testnet4]
server=1
rpcuser=username
```